app/scripts/lib/sentry.js

```diff
@@ -0,0 +1,133 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+define([
+  'underscore',
+  'raven',
+  'lib/url'
+], function (_, Raven, Url) {
+
+  var ALLOWED_QUERY_PARAMETERS = [
+    'client_id',
```

Comment:
This might sound silly because we haven't done it very much, can you alphabetize these so that we know where to look for an item if doing a quick scan?